crates/ln-dlc-node/src/tests/dlc_channel.rs

```diff
@@ -166,8 +168,17 @@ async fn can_open_and_collaboratively_close_channel() {
 
     tracing::debug!("Accepting collaborative close offer");
 
-    app.accept_dlc_channel_collaborative_close(&coordinator_signed_channel.channel_id)
-        .unwrap();
+    let offered_payout = match coordinator_signed_channel.state {
+        SignedChannelState::Settled { counter_payout, .. } => counter_payout,
+        _ => panic!("Only a channel in state Settled can be closed collaboratively"),
```

Comment:
:upside_down_face: I think that's not strictly true, but we only want to let settled channels be closed collaboratively.